subsys/bluetooth/host/cs.c

```diff
@@ -617,7 +617,7 @@ int bt_le_cs_start_test(const struct bt_le_cs_test_param *params)
 
 	cp->override_parameters_length = override_parameters_length;
 
-	struct bt_hci_cmd_hdr *hdr = (struct bt_hci_cmd_hdr *)buf->data;
+	struct bt_hci_cmd_hdr *hdr = (struct bt_hci_cmd_hdr *)&buf->data[1];
```

Comment:
Suggest to add a comment explaining why it's offset by 1. 

Is there a way that we can perhaps make this a relative offset, so that if the buffer header changes (although unlikely), we don't have to modify this harcoded offset?

Reply:
I thought that there is e.g. an API or a constant in the host that I could use instead of `1` but nothing like that was added with #88710 . Aim of this PR was to make a quick fix. 

I propose host maintainers to add such helpers is in host and maybe use it instead of `buf->data[0]` and `buf->data[1]` in host.

Reply:
I created a draft PR implementing the above proposal here: #91865

Reply:
couldn't we apply both changes? Or do you believe https://github.com/zephyrproject-rtos/zephyr/pull/91865 will be quick?

Reply:
> couldn't we apply both changes so that we can have a simple, temporary workaround without needing to deal with API deprecation?

The difference would in practice be the difference in the timing of opening the PRs, so one day, since CI is happy with the other PR and there weren't any objections in the Bluetooth WG meeting where I did an overview. So the "temporary" in your comment is in practice 24 hours or so. I'm not sure I understand your point about API deprecation, since that'll happen either way once the other PR goes in?

So I'm not sure it's worth the hassle of rebasing and fixing conflicts if we try to merge both. What I would appreciate though, is if you could do some basic testing of the other PR, in case there's something that CI didn't catch.